R/mod_packageDependencies.R

```diff
@@ -0,0 +1,336 @@
+                             list(
+                               score = formattable::formatter(
+                                 "span",
+                                 style = x ~ formattable::style(display = "block",
+                                                                "border-radius" = "4px",
+                                                                "padding-right" = "4px",
+                                                                "font-weight" = "bold",
+                                                                "color" = "white",
+                                                                "order" = x,
+                                                                "background-color" = formattable::csscolor(
+                                                                  setColorPalette(100)[round(as.numeric(x)*100)]))),
+                               decision = formattable::formatter(
+                                 "span",
+                                 style = x ~ formattable::style(display = "block",
+                                                                "border-radius" = "4px",
+                                                                "padding-right" = "4px",
+                                                                "font-weight" = "bold",
+                                                                "color" = ifelse(x %in% decision_lst, "white", "inherit"),
+                                                                "background-color" = 
+                                                                  ifelse(x %in% decision_lst, 
+                                                                         color_lst[x], 
+                                                                         "transparent")))
+                             )),
+                           selection = list(mode = 'multiple'),
+                           colnames = c("Package", "Type", "Name", "Version", "Score", "Review Package"),
+                           rownames = FALSE,
+                           style="default"
+                         ) %>%
+                           DT::formatStyle(names(my_data_table()), textAlign = 'center')
+                       })
+                ),
+                column(width = 4, 
+                       style="position:relative; top: 0px; right 0px; left: 250px;",
+                       # renderCachedPlot({
+                       #   deepdep::plot_dependencies(dd(), type = "circular", same_level = TRUE, show_version = TRUE, reverse = TRUE, show_stamp = FALSE)
+                       # }, cacheKeyExpr = { list(dd()) },
+                       # sizePolicy = sizeGrowthRatio(width = 900, height = 900) )
+                       renderPlot(width = 900, height = 900, {
+                         deepdep::plot_dependencies(dd(), type = "circular", same_level = TRUE, show_version = TRUE, reverse = TRUE, show_stamp = FALSE)
```

Comment:
It feels like one of the ideas here is to fit the plot on page to the right of the table. I was able to achieve this in another project. The trick was finding out how wide the space was available based on the loaded window. Happy to share if you want.

Reply:
Issue #550